--- origsrc/gts-snapshot-120706/src/Makefile.am	2012-07-06 06:27:04.000000000 -0500
+++ src/gts-snapshot-120706/src/Makefile.am	2012-08-24 00:16:51.351494400 -0500
@@ -65,11 +65,11 @@ include_HEADERS = \
 predicates.o: predicates.c predicates_init.h predicates.h
 	$(COMPILE) -c $(srcdir)/predicates.c
 
-predicates_init: predicates_init.c rounding.h
-	$(COMPILE) $(srcdir)/predicates_init.c -o $(srcdir)/predicates_init
+predicates_init$(EXEEXT): predicates_init.c rounding.h
+	$(COMPILE) $< -o $@
 
-predicates_init.h: predicates_init
-	./predicates_init >  $(srcdir)/predicates_init.h
+predicates_init.h: predicates_init$(EXEEXT)
+	./predicates_init > $@
 
 CLEANFILES = $(BUILT_SOURCES)
 
